wordpress/wp-content/plugins/sil-dictionary-webonary/webonary/Webonary_Cloud.php

```diff
@@ -127,8 +127,15 @@ public static function entryToReversal($dictionaryId, $lang, $letter, $entry) {
 		//<div class=post><div xmlns="http://www.w3.org/1999/xhtml" class="reversalindexentry" id="g009ab666-43dd-4f2f-ba62-7017417f6b23"><span class="reversalform"><span lang="en">aardvark</span></span><span class="sensesrs"><span class="sensecontent"><span class="sensesr" entryguid="gee1142ec-65f5-4e23-8d95-413685a48c23"><span class="headword"><span lang="mos"><a href="https://www.webonary.org/moore/gee1142ec-65f5-4e23-8d95-413685a48c23">tãnturi</a></span></span><span class="scientificname"><span lang="en">orycteropus afer</span></span></span></span></span></div></div>
 		$id = self::convertGuidToId($entry->_id);
 		$reversal_value = '';
-		foreach ($entry->senses->definitionOrGloss as $definition)	{
-			if (($lang == $definition->lang) && ($letter == substr($definition->value, 0, 1))) {
+
+		$definitions = $entry->senses->definitionOrGloss;
+		if (!is_array($definitions)) {
+			$definitions = [$definitions];
+		}
+
+		foreach ($definitions as $definition)	{
```

Comment:
Looks like there's a tab instead of a space between ")" and "{".

Reply:
good eye :-) removed the tab